kuma/wiki/jinja2/wiki/document.html

```diff
@@ -60,8 +60,10 @@
 {% block site_css %}
     {{ super() }}
     {% if zone_stack %}
-      {% for zone in zone_stack|reverse() %}
-        <link rel="stylesheet" type="text/css" href="{{ url('wiki.styles', zone.document.slug) }}">
+      {% for zone in zone_stack|reverse() if zone.css_filename %}
+        {% stylesheet zone.css_filename %}
+      {% else %}
+        {% stylesheet 'zones' %}
```

Comment:
I'm not sure this is valid Jinja2 code.  You may need to split it into

```
{% if zone.css_filename %}
   {% for zone in zone_stack|reverse() %}
       {% stylesheet zone.css_filename %}
   {% endfor %}
{% else %}
   {% stylesheet 'zones' %}
{% endif %}
```
But, I could be wrong - we'll see when tests are working.